definitions/features/foreman_cockpit.rb

```diff
@@ -0,0 +1,15 @@
+class Features::ForemanCockpit < ForemanMaintain::Feature
+  metadata do
+    label :foreman_cockpit
+
+    confine do
+      find_package('tfm-rubygem-foreman_remote_execution-cockpit')
```

Comment:
We should only have this feature when foreman_server is available ?

Reply:
Yeah, that would be an extra check. Added it.